Games/LLEGSB/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style>
```

Comment:
Typically, we put all this style in the `css` file rather than clutter up the html with it.